data/coalition/coalition missions.txt

```diff
@@ -333,7 +333,7 @@ mission "Coalition Yottrite 3"
 				`	"That's gotta be the shortest courier mission I've ever heard of. Can't some other ship do the trip?"`
 					goto soitwouldseem
 			label insidecrate
-			`	"Older samples that you've brought to us, they contain. Experimented with, the Yottrite was. Unsure of the details, I am."`
+			`	"Older samples that you've brought to us, they contain. Experimented with, the yottrite was. Unsure of the details, I am."`
```

Comment:
```suggestion
			`	"Older samples that you've brought to us, they contain. Experimented with, the yottrite was. Unsure of the details, I am."`
```

I always figured Yottrite was capitalized specifically as a proper name for what it is, but probably best to defer to how @Arachi-Lover wants to go with that for consistency, as otherwise it -is- on par with capitalizing every reference to Silver or Gold, and asteroid materials probably should be an exception, since the "outfit" itself is already a direct representation of their general form instead of any specific item.

Reply:
As far as I know, the coalition missions are the only place where minerals are capitalized.

Reply:
And only Yottrite specifically, right? Just wanting to check if that was a deliberate choice or not.

I do -not- recommend capitalizing all mineral names in general. :)

Reply:
Yes. Only yottrite, and only there. I don't think they mention any other minerals in those missions tbh.

Reply:
Yeah seems it's not capitalized in nearly all but 2 cases I found, one being this.
If the way we're going is to not capitalize minerals, then can uncapitalize those cases too.